### PR TITLE
Rewording README and splitting it into Wiki

### PR DESCRIPTION
**Pull request by meshuggahtas**
This commit only consists the README.md change.

The README.md have been reworded and some words have been changed with synonyms.

My Wiki with 3 pages has the remaining information as nicely as I could format it.
https://github.com/meshuggahtas/tasvideos/wiki

Please comment on my wiki as well in this PR so we can discuss its content if there is a problem with it easier.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,60 +1,23 @@
 ![.NET Core](https://github.com/TASVideos/tasvideos/workflows/.NET%20Core/badge.svg)
 
 # TASVideos 2.0
-This is a rewrite attempt of the tasvideos website.
+Rewrite of the original tasvideos website.
 
-It aims to change technologies, have simpler more maintainable code, and bring the site up to modern standards.
+Aims to bring the site up to modern standards with updated technologies and a more maintainable and managable code.
```
[the remark this answers is not in the document]
Comment (meshuggahtas) [reply README.md]: Fixed.

[the remark this answers is not in the document]
Comment (meshuggahtas) [reply README.md]: Fixed

**Comment by TiKevin83** [comment]
sounds good to me to have this info maintained on the wiki, can you update contributing.md to point to the desired wiki resources?

**Comment by meshuggahtas** [comment]
Updated. I will copy paste the source of the (currently 4) wiki pages from my fork to the tasvideos wiki after the pull request have been merged and its content agreed upon.

**Review by adelikat — APPROVED** [review]

**Comment by YoshiRulz** [comment]
Under https://github.com/TASVideos/tasvideos/wiki/Local-Development-Setup#ubuntu-20043 you've used `<code/>` (single backtick in Markdown) instead of `<pre/>` (triple-backtick fences). The latter also supports [syntax highlighting](https://docs.github.com/en/github/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks#syntax-highlighting) FWIW.

**Comment by meshuggahtas** [comment]
Thanks for the tip! Added bash and \ to the end of lines for multiline commands.

**Comment by meshuggahtas** [comment]
Added link https://github.com/TASVideos/tasvideos/wiki/Local-Development-Setup to the end of README.md.

**Pull request merged**